test/parallel/test-inspector-console-top-frame.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const common = require('../common');
```

Comment:
Please add a description for the test

```suggestion
// Many Inspector protocol clients rely on the top frame reported for the
// console messages. This test makes sure correct location is reported.

const common = require('../common');
```